src/mavlink/mavlink_camera.rs

```diff
@@ -695,7 +767,13 @@ fn receive_message_loop(
                             continue;
                         }
 
-                        let information = mavlink_camera_information.lock().unwrap();
+                        let information = match mavlink_camera_information.lock() {
+                            Ok(guard) => guard,
+                            Err(error) => {
+                                error!("Failed locking a Mutex. Reason: {error}");
+                                return;
+                            }
+                        };
```

Comment:
We should create a macro for this

Reply:
That's true. I should start writing small macros like that